### DISCUSSION TITLE
RaspberryMatic als HA Add-on und HB-RF-ETH

### DISCUSSION THREAD

**Discussion by marc965** [OpenCCU/OpenCCU#1465] [🇩🇪 Fragen&Antworten]
Hallo.

Habe seit einigen Wochen einen Raspberry Pi4 mit direkt aufgesteckter RPI-RF-MOD Karte und piVCCU3 im Einsatz. Parallel läuft eine Home Assistant Installation und kommuniziert erfolgreich mit der Homematic.

Seit Montag versuche ich nun meine frisch gekaufte HB-RF-ETH Platine zu betreiben. Der Raspberry Pi4 wurde abgeschaltet, die bereits vorhandene RPI-RF-MOD Karte aufgesteckt und in Betrieb genommen. Auf der Home Assistant Installation läuft nun zusätzlich das RaspberryMatic Addon und kommuniziert mit der HB-RF-ETH:

![image](https://user-images.githubusercontent.com/85791333/138343672-2e4fe754-0532-49ab-b9ef-c3c8fb347098.png)

### Auf HA Seite das RaspberryMatic Protokoll sieht auch nicht schlecht aus:
Mounting /data as /usr/local (Home Assistant Add-On): OK
Identifying onboard hardware: oci, OK
Initializing RTC Clock: onboard, OK
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Starting logging: OK
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.6, OK
Identifying Homematic RF-Hardware: ......HmRF: RPI-RF-MOD/HB-RF-ETH@192.168.133.110, HmIP: RPI-RF-MOD/HB-RF-ETH@192.168.133.110, OK
Updating Homematic RF-Hardware: RPI-RF-MOD: 4.2.14, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting sshd: OK
Starting ha-proxy: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: .OK
Starting rfd: .OK
Starting HMIPServer: ........OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: disabled
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK

In der RaspberryMatic WebGUI sieht auch alles normal aus, das zuvor auf der piVCCU3 angemeldete HmRF Gerät ließ sich auch sofort anlernen und war auch in HA erreichbar.

Allerdings bleiben die LEDs der  RPI-RF-MOD Platine dunkel, es leuchtet nur die HB-RF-ETH grün und es lassen sich keine HmIP Geräte anlernen.

Mir ist nicht klar was nun nicht richtig funktioniert, das Anlernen selbst, die RaspberryMatic oder die HB-RF-ETH.
Da ich mit der Homematic erst beginne meine Wohnung auszustatten ist die Anzahl der Geräte (noch) überschaubar:
1. HM-ES-PMSw1-PI-DN-R1 (HmRF) lässt sich anlernen
2. 2 x HmIP-eTRV-C-2 lassen sich nicht anlernen
3. 2 x HmIP-WTH-2 lassen sich nicht anlernen

Ich habe es über beide Methoden versucht, mit und ohne Internetzugang. Habe alle Geräte auch resettet.
Irgendwie lassen sich die Geräte auch jetzt nicht mehr in den Anlernmodus versetzen (System Taste länger drücken).
Komisch ist, dass bei den HmIP-WTH-2 Geräten das Funksymbol im Display blinkt, als wäre keine Zentrale erreichbar.
Das war mit der piVCCU3 nicht...

Welche Logs habe ich auf der RaspberryMatic die ich einsehen könnte?

Folgende Versionen sind im Einsatz:
1. HB-RF-ETH Firmware v1.2.9
2. Home Assistant OS 6.5, supervisor-2021.10.0, core-2021.10.6
3. RaspberryMatic 3.59.6.20211009
4. Synology DS1520+, DSM 7.0-41890, Virtual Machine Manager 2.5.4-11440

Hat jemand einen Tipp? Was mache ich falsch?

Grüße
Marc

**Comment by marc965** [comment]
Aktualisierung: 22.10.2021
Habe nun separat eine RaspberryMatic direkt auf der Synology aufgesetzt (RaspberryMatic-3.59.6.20211009.ova).
Die HB-RF-ETH wird wohl erkannt, die blaue LED der RPI-RF-MOD leuchtet (was sie nicht tat als die RaspberryMatic unter HA lief).
Leider gleiches Problem: HmRF Gerät lässt sich einwandfrei verbinden, HmIP Geräte Anbindung erfolglos...

Ich bin ratlos...

**Comment by marc965** [comment]
Neuer Test:
1. Wiederinbetriebnahme der alten piVCCU3: Alle Geräte konnten erfolgreich neu angelernt werden.
2. Geräte auf der piVCCU3 erfolgreich "abgelernt"
3. piVCCU3 außer Betrieb genommen
4. Wahlweise erneutes Hochfahren der RaspberryMatic Appliance oder HA Addon: Nur HmRF Gerät lässt sich anlernen

Also liegt kein Hardware Defekt vor, alle Geräte arbeiten einwandfrei, es fehlt lediglich in der virtuellen Umgebung etwas...

**Reply by jens-maus** [reply]
Tut mir leid, aber hier in den Testumgebungen geht das. Da HmIP Listen-before-talk macht kann es auch einfach sein das dein Rauschen in der Umgebung des Funkmodules am HB-RF-ETH zu hoch ist und deshalb der Empfang/Sende zu grenzlastig ist das HmIP problemlos geht.

**Comment by marc965** [comment]
Hallo.
Danke für die Antwort.
Das verstehe ich nicht. Mit dem RaspberryPi4 und dem gleichen Funkmodul am gleichen Ort funktioniert es.
Die HB-RF-ETH Platine ändert daran doch nichts, oder?
Und das Wandthermostat habe ich ganz in der Nähe stehen (ca. 3 Meter entfernt).
Und warum leuchtet die blaue Lampe nicht.
Ist HmRF weniger anfällig für Störsignale?

Irgendwelche Tipps wie ich der Sache auf den Grund gehen kann?

Viele Grüße
Marc

PS: Wie sieht denn deine Testumgebung aus?

**Comment by marc965** [comment]
Hallo Jens!

Ich habe gerade eben die Homematic wieder zurückgebaut auf die Paspberry Pi4 basierte piVCCU3 Version.
![image](https://user-images.githubusercontent.com/85791333/138432549-cb222a3a-28e7-4171-a702-38d36f37d13c.png)

Wie du auf dem Bild sehen kannst ist unten der Paspberry Pi4 mit der aufgesteckten RPI-RF-MOD in Betrieb.
Oben ist die HB-RF-ETH verbaut.
Ich habe lediglich die RPI-RF-MOD Platine von der HB-RF-ETH oben auf den Raspberry unten umgesteckt,
Innerhalb von einer Stunde hatte ich meine Heizungssteuerung auf der piVCCU3 wieder am laufen.
Das Anlernen klappt sofort problemlos.

Das kann doch nicht an Störsignalen liegen.
Da muss ich noch irgendein Detail übersehen haben.
Kann das an der Synology liegen?

Welche Möglichkeiten habe ich auf dem HA Addon oder der RaspberryMatic Appliance in Logfiles nach dem Fehler zu fahnden?
Warum leuchtet beim HA Addon die blaue LED nicht, während sie mit der Appliance funktioniert?

Viele Grüße
Marc
